### PR TITLE
Push 923&925 commits from rls/daal-2020-mnt to master

### PR DESCRIPTION
**Pull request by OnlyDeniko**

### CODE REVIEW

**Review by agorshk — COMMENTED** [review]
please add changes from [PR](https://github.com/oneapi-src/oneDAL/pull/925) related with fix on vs2017, the changes is needed on 590 row of df_regression_train_dense_default_impl.i file

**Comment by agorshk** [comment]
2 comments were removed for decreasing resources on wrong PR

File: cpp/daal/src/algorithms/dtrees/forest/classification/df_classification_train_dense_default_impl.i [hunk 1]
```diff
@@ -476,48 +490,87 @@ bool UnorderedRespHelper<algorithmFPType, cpu>::findBestSplitCategoricalFeature(
...
+{
+    const IndexedFeatures::IndexType * const indexedFeature = this->indexedFeatures().data(iFeature);
+    const auto aResponse                                    = this->_aResponse.get();
+    const auto aWeights                                     = this->_aWeights.get();
+
+    auto nFeatIdx         = _idxFeatureBuf.get();
+    auto featWeights      = _weightsFeatureBuf.get();
+    auto nSamplesPerClass = _samplesPerClassBuf.get();
 
     algorithmFPType bestImpDecrease =
```
Comment (agorshk) [inline cpp/daal/src/algorithms/dtrees/forest/classification/df_classification_train_dense_default_impl.i:548]: remove it, have a look original [PR](https://github.com/oneapi-src/oneDAL/pull/923) (please check that library was successfully built locally before push update)

Comment (OnlyDeniko) [reply cpp/daal/src/algorithms/dtrees/forest/classification/df_classification_train_dense_default_impl.i]: Corrected

**Review by agorshk — APPROVED** [review]
check CI before merge

**Pull request merged**